### ISSUE TITLE
Exclude on edges option?

### ISSUE THREAD

**Issue by n-amb** [MouseLand/cellpose#249]
Hi Cellpose community,  

I am really pleased with the Cellpose segmentation results and I look forward to using it in the future. To this end, I would interested to know if there is an "exclude on edges" feature, similar to that in ImageJ, within the current release? If not, is there a work-around within the Cellpose to LabelsToROI plugin pipeline that would accomplish something similar? 

Thanks!

**Comment by kevinjohncutler** [comment]
This is a good idea - for now it would be pretty simple to do as a post-processing step in a Jupyter notebook, but it would be great to implement (maybe under utils or transforms) as an optional flag. It would also be very useful for breaking apart large images to be stitched later.

**Comment by carsen-stringer** [comment]
certainly doable, to be clear this removes masks touching the edge of the image?

hmm for stitching you may still get duplicates that aren't touching the edges, but I guess for a rough count that's fine? I'm not sure the optimal stitching method to be honest.

**Comment by n-amb** [comment]
@carsen-stringer Yes, the exclude on edges option would remove masks touching the edge of images.  For our purposes, this would avoid segmenting cells/objects that are only partially in view.  Like @kevinjohncutler suggested we ended up editing the mask files as a post-processing step, but an optional flag is a perhaps a better idea. Thanks!

**Comment by carsen-stringer** [comment]
this has been added as a command line flag in the github version, try `--exclude_on_edges`

**Comment by meahgem** [comment]
Hello,
Thank you for cellpose segmentation. It is saving me a lot of time with analysis.

I tried using the exclude_on_edges for segmentation 

model = models.Cellpose(gpu=True, model_type='cyto')
channels = [0,0]
masks, flows, styles, diams = model.eval(anno_imgs, diameter=None, channels=[0,0], flow_threshold=0.4, do_3D=False, exclude_on_edges=True)

but got an error
eval() got an unexpected keyword argument 'exclude_on_edges'

Is it possible to use the function in jupyter lab?

Thank you
Megha
